StarLight_Line/GameLib/DX/DXInput/InputDev/Keyboard/Keyboard.cpp

```diff
@@ -0,0 +1,41 @@
+﻿#include "Keyboard.h"
+
+#include <windows.h>
+
+#include <dinput.h>
+
+VOID Keyboard::Create(LPDIRECTINPUT8 pDXInput)
+{
+	pDXInput->CreateDevice(
+				GUID_SysKeyboard, &m_pDInputDev,
+				NULL);
+
+	m_pDInputDev->SetDataFormat(&c_dfDIKeyboard);
+
+	m_pDInputDev->SetCooperativeLevel(
+					m_HWND,
+					DISCL_NONEXCLUSIVE | DISCL_BACKGROUND);
+}
+
+VOID Keyboard::CheckInputStateDetatils()
```

Comment:
自分のコメント読み返してて「タイポしたか？！」と思ったのですが、コピー元が間違っていたようです。
```suggestion
VOID Keyboard::CheckInputStateDetails()
```
ですね。